src/Alien.js

```diff
@@ -0,0 +1,42 @@
+import { colliding } from './colliding.js/'
+import { context } from './constants.js'
+import { getRandomColor } from './randomcolor'
+class Alien {
+  constructor (game) {
+    this.center = {
+      x: Math.floor(Math.random() * 500),
+      y: 20
+    }
+    this.size = {
+      x: 20,
+      y: 20
+    }
+    this.game = game
+    this.color = getRandomColor()
+    this.speed = Math.random() * 1
+  }
+  draw () {
+    context.fillStyle = this.color
+    context.fillRect(this.center.x, this.center.y, this.size.x, this.size.y)
+  }
+  update () {
+    this.center.y += this.speed
+    if (this.center.y >= 500) {
+      this.game.gameOver = true
+    }
+    this.game.bullets.forEach((bullet, bulletIndex) => {
```

Comment:
I dig this section -- I like how you're checking for collisions and also updating the number of aliens and checking for the winner at once.